<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scorecanvas</class>
 <widget class="QMainWindow" name="scorecanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scoreCanvas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawDiscreteNotes"/>
   <addaction name="actionDrawContinuousNotes"/>
   <addaction name="actionDrawPhrase"/>
   <addaction name="separator"/>
   <addaction name="actionRecordDiscreteNotes"/>
   <addaction name="actionRecordContinuousNotes"/>
   <addaction name="separator"/>
   <addaction name="actionScoreCanvasSelect"/>
   <addaction name="actionScoreCanvasZoom"/>
  </widget>
  <widget class="QToolBar" name="scoreCanvasBottomToolbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionstop"/>
  </widget>
  <action name="actionDrawDiscreteNotes">
   <property name="text">
    <string>Draw Discrete Notes</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawContinuousNotes">
   <property name="text">
    <string>Draw Continuous Notes</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawPhrase">
   <property name="text">
    <string>Draw Phrase</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRecordDiscreteNotes">
   <property name="text">
    <string>Record Discrete Notes</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRecordContinuousNotes">
   <property name="text">
    <string>Record Continuous Notes</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRecordPhrase">
   <property name="text">
    <string>Record Phrase</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionScoreCanvasSelect">
   <property name="text">
    <string>Select</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionScoreCanvasZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>▶</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>■</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
